proposals/78:upgrade-17/localchain.test.js

```diff
@@ -0,0 +1,33 @@
+import test from 'ava';
+
+import { agd, evalBundles, waitForBlock } from '@agoric/synthetic-chain';
+
+const SUBMISSION_DIR = 'localchaintest-submission';
```

Comment:
I think we need this directory populated with files that we built with `a3p-integration/scripts/build-all-submissions.sh`.

Reply:
Confirmed; CI got some expected errors:
```
PPT post {
  totalMintedConverted: { brand: '$0.Alleged: IST brand', value: '+20010000' },
  totalMintedProvided: { brand: '$0', value: '+2750000' },
  walletsProvisioned: '+11'
}
  ✘ [fail]: provisionPool › upgrading provisionPool and vat-bank Rejected promise returned by test
    ℹ add assets before
    ℹ upgrade Bank
  ─

  localchain › localchain passes tests

  Rejected promise returned by test. Reason:

  Error {
    code: 'ENOENT',
    errno: -2,
    path: 'localchaintest-submission',
    syscall: 'scandir',
    message: 'ENOENT: no such file or directory, scandir \'localchaintest-submission\'',
  }



  provisionPool › upgrading provisionPool and vat-bank
  Rejected promise returned by test. Reason:

  Error {
    code: 'ENOENT',
    errno: -2,
    path: 'upgrade-bank',
    syscall: 'scandir',
    message: 'ENOENT: no such file or directory, scandir \'upgrade-bank\'',
  }
```